e2e-tests/demand-backup-physical/run

```diff
@@ -80,124 +82,47 @@ compare_mongo_cmd "find" "myApp:myPass@${cluster}-rs0-2.${cluster}-rs0.${namespa
 
 echo 'running backups'
 backup_name_minio="backup-minio"
+run_backup minio ${backup_name_minio}
 if [ -z "$SKIP_BACKUPS_TO_AWS_GCP_AZURE" ]; then
 	backup_name_aws="backup-aws-s3"
 	backup_name_gcp="backup-gcp-cs"
 	backup_name_azure="backup-azure-blob"
-fi
 
-if [ -z "$SKIP_BACKUPS_TO_AWS_GCP_AZURE" ]; then
 	run_backup aws-s3 ${backup_name_aws}
 	run_backup gcp-cs ${backup_name_gcp}
 	run_backup azure-blob ${backup_name_azure}
-fi
-run_backup minio ${backup_name_minio}
 
-if [ -z "$SKIP_BACKUPS_TO_AWS_GCP_AZURE" ]; then
 	wait_backup "${backup_name_aws}"
 	wait_backup "${backup_name_gcp}"
 	wait_backup "${backup_name_azure}"
 fi
 wait_backup "${backup_name_minio}"
 
-echo "drop collection"
-run_mongo 'use myApp\n db.test.drop()' "myApp:myPass@${cluster}-rs0.${namespace}"
-
 if [ -z "$SKIP_BACKUPS_TO_AWS_GCP_AZURE" ]; then
+	echo "drop collection"
+	run_mongo 'use myApp\n db.test.drop()' "myApp:myPass@${cluster}-rs0.${namespace}"
```

Comment:
maybe change echo to desc? As @hors did in this PR https://github.com/percona/percona-server-mongodb-operator/pull/1142

Reply:
@tplavcic @nmarukovich In general, everything is ok, but we need to decide what we should do with #1142. We need to adapt these changes in the same way as I did it in #1142 or mege it and then rework. But I think we will have a lot of conflicts with #1142

Reply:
@hors I will merge this one and we can discuss about #1142 , I have left some comments there, but anyway whatever changes are needed for backup tests they can be done there.